### DISCUSSION TITLE
How to Troubleshoot no Data in Dashboard

### DISCUSSION THREAD

**Discussion by cocjos** [Security-Onion-Solutions/securityonion#13077] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

16 gb

### Storage for /

1TB

### Storage for /nsm

1TB

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I do not see any log in information in the dashboard. I am trying to connect windows elastic agents to the SO manager. I see the agents in fleet manager. The agents are connected to SO manager when I look at the agents' netstat. 

I see this error  Failed to send backlog of events to Redis in logstash.log. But I am not sure what I am looking at yet.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Looks like an issue with the Logstash pipeline. Would you restart Logstash, then re-check the /opt/so/log/logstash/logstash.log for any clues.

Also, in Fleet, do you see all the agents as healthy?

**Comment by cocjos** [comment]
Chris,

I do see all the agents in fleet. The agents are all good.

Joseph Lamb
________________________________

**Reply by cm-ops** [reply]
Restart Logstash, tail the` /opt/so/log/logstash/logstash.log`, look for any issues with the pipeline(s) starting.

**Comment by cocjos** [comment]
Chris,

The pipeline is not starting workers.

This is the start:

[2024-05-29T13:58:57,068][WARN ][logstash.outputs.redis   ] Failed to send backlog of events to Redis

 "/usr/share/logstash/logstash-core/lib/logstash/outputs/base.rb:104:in `multi_receive'", "org/logstash/config/ir/c                    ompiler/AbstractOutputDelegatorExt.java:121:in `multi_receive'", "/usr/share/logstash/logstash-core/lib/logstash/java_pipeline.rb:304:in `block in st                    art_workers'"]}

________________________________

**Reply by cm-ops** [reply]
See if you see a pipeline terminated in the log, also check your /opt/so/log/elasticsearch/securityonion.log for any issues. 

How is the memory utilization? Another issue could be a Logstash heap issue shutting down the pipeline(s).

**Reply by dougburks** [reply]
@DrOnionMaster Please review the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion.

**Reply by DrOnionMaster** [reply]
My apologies, comments deleted.
